### ISSUE TITLE
Error while building Hello-Plugin

### ISSUE THREAD

**Issue by Hrushi20** [second-state/wasmedge_plugin_rust_sdk#2]
![Build Error](https://user-images.githubusercontent.com/65367566/223467713-56fbae68-11ba-4606-b9e6-40a2e2a4797b.png)

Looks like I'm missing the required `C` header files to build the plugins. How do I add the header files for [wasmedge_sys_ffi](https://github.com/second-state/wasmedge_plugin_rust_sdk/tree/main/wasmedge_sys_ffi)? 

```
rustup version: 1.25.2
cargo version: 1.67.1
Os: Ubuntu 22.04 LTS
```

**Comment by L-jasmine** [comment]
Have you installed wasmedge？And what is WasmEdge install Path？

**Comment by Hrushi20** [comment]
I have installed wasmedge. The path is `~/.wasmedge/bin/wasmedge`

**Comment by L-jasmine** [comment]
ok，you should execute `export WASMEDGE_LIB_DIR={path that include wasmedge.so}`.
sorry, we need update Wasmedge install.sh

**Comment by L-jasmine** [comment]
Do you have a directory like `~/.wasmedge/plugins` ? 
You should move your `libhello_plugin.so` into that.

**Comment by Hrushi20** [comment]
> ok，you should execute `export WASMEDGE_LIB_DIR={path that include wasmedge.so}`. sorry, we need update Wasmedge install.sh

Do I need add it to the Cargo.toml file in the Hello Plugin?

 


> Do you have a directory like `~/.wasmedge/plugins` ? You should move your `libhello_plugin.so` into that.

Yes, I do have the above directory. Thank You

**Comment by L-jasmine** [comment]
like this.
```sh
# examples/plugin/hello_plugin
export WASMEDGE_LIB_DIR={path that include wasmedge.so}
cargo build --release
```

**Comment by Hrushi20** [comment]
Thank You. This fixed the issue :smile:

**Comment by L-jasmine** [comment]
In wasmedge 0.11.2, C-Plugin have a bug.
https://github.com/WasmEdge/WasmEdge/issues/2343
This crate is base on C-Plugin.
So when you want to try other example plugin, please delete old `plugin.so` .
This bug is fixed on 0.12

**Comment by Hrushi20** [comment]
I'll ensure to delete the old `plugin.so`. Thanks.
